website/src/Pages/GalleryPage/MediaRenderer.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import { Box } from '@mui/material';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function MediaRenderer({ mediaArray }) {
+  return (
+    <Box>
+      {mediaArray.map((media, index) => {
+        const contentType = media.ContentType.slice(0, 5);
+        const mediaUrl = media.MediaURL;
+
+        if (contentType === 'image') {
+          return <img style={{ height: 300, width: 300 }} src={mediaUrl} alt={`${index}`} />;
+        } if (contentType === 'audio') {
+          return (
+            <audio style={{ height: 300, width: 300 }} controls>
+              <source src={mediaUrl} type="audio/mpeg" />
+            </audio>
+          );
+        } if (contentType === 'video') {
+          return (
+            <video style={{ height: 300, width: 300 }} controls>
+              <source src={mediaUrl} type="video/mp4" />
+            </video>
+          );
+        }
+
+        return null;
```

Comment:
might be worth it to have a text field that says "invalid media type: [media type]" if we don't get image/audio/video so that admins can review it as needed and don't accidentally approve something that has an extra file that can't be rendered or something

Reply:
actually that gets avoided by Ryan's upload feature since it requires a certain type to upload by S3, so I don't think that's necessary